src/main/java/com/chessix/sepa/pain/impl/DocumentServiceImpl.java

```diff
@@ -49,8 +49,8 @@ public void generatePain00800202(OutputStream outputStream, Creditor creditor, I
     }
 
     @Override
-    public String generatePain00800102(Creditor creditor, InitiatingParty initiatingParty, FirstTransactions firstTransactions, RecurringTransactions recurringTransactions, boolean useBatchBooking, Dialect dialect) {
-        Pain00800102 pain00800102 = new Pain00800102(creditor, initiatingParty, firstTransactions, recurringTransactions, useBatchBooking, dialect);
```

Comment:
It looks like this changes the function instead of creating an extra one. Will this still support non-batching mode?

Reply:
Maybe it is time to let go of Java 8 by now :-) The API incompatibility should be a simple fix for anyone using the library so I think we should keep this change. Thanks!